test/apps/test_init_mt.c

```diff
@@ -50,6 +56,8 @@ int main(int argc, char **argv)
 
 #pragma omp barrier
 
-    printf("finished %d threads\n", count);
-    return 0;
+    duration = time(NULL) - start;
+    printf("%sfinished %d threads after %ld seconds\n",
```

Comment:
maybe print timestamp for start and finish (using gettimeofday) so will see if test is stuck?
BTW this could also be done in MTT runner

Reply:
added timestamps

Reply:
can unite the print with line 65

Reply:
merged into single call